src/components/TeacherViews/Onboarding/HowCanOakSupport/HowCanOakSupport.view.tsx

```diff
@@ -107,18 +107,21 @@ const HowCanOakSupport = () => {
               control={control}
               name={key as OakSupportKey}
               key={key}
-              render={({ field }) => (
-                <OakCheckBox
-                  id={key}
-                  key={key}
-                  {...field}
-                  value={value}
-                  onChange={(event) => {
-                    clearErrors();
-                    field.onChange(event);
-                  }}
-                />
-              )}
+              render={({ field }) => {
+                const { ref, ...fieldProps } = field;
```

Comment:
@njblore turns out we were passing `ref` here but [`<OakCheckBox/>`](https://github.com/oaknational/oak-components/blob/main/src/components/molecules/OakCheckBox/OakCheckBox.tsx) doesn't use [`forwardRef(...)`](https://react.dev/reference/react/forwardRef)